src/renderer/components/ft-toast/ft-toast.js

```diff
@@ -0,0 +1,53 @@
+import Vue from 'vue'
+import FtToastEvents from './ft-toast-events.js'
+
+export default Vue.extend({
+  name: 'FtToast',
+  data: function () {
+    return {
+      toasts: [
+        { isOpen: false, message: '', action: null, timeout: null },
```

Comment:
Now messages are cleared from the renderer once all messages have been hidden. If they were to be removed as soon as they are hidden, it would lead to the toasts being re-rendered on the page causing them to flicker and change position, this would be annoying if a user went to click on a toast but it was moved to the bottom of the list. It also waits 300ms before attempting to clear them so that the last one has enough time to fade out.